web/src/components/apps/DashboardGraphsCard.jsx

```diff
@@ -3,7 +3,7 @@ import dayjs from "dayjs";
 import localizedFormat from "dayjs/plugin/localizedFormat"
 import Handlebars from "handlebars";
 import Modal from "react-modal";
-import { getValueFormat } from "@grafana/ui"
+import { getValueFormat } from "@grafana/data"
```

Comment:
These look like unrelated changes

Reply:
It is related actually, when running yarn install, we were using a very outdated version of grafana/ui causing getValueFormat to not be defined. In the latest grafana/ui, we need grafana/data for the getValueFormat function. https://grafana.com/docs/grafana/latest/packages_api/data/getvalueformat/
I should've added a review comment to explain this earlier.

Reply:
Gotcha thanks for that context